crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -47,15 +49,16 @@ impl DynamicScene {
             DynamicSceneBuilder::from_world_with_type_registry(world, type_registry.clone());
 
         builder.extract_entities(world.iter_entities().map(|entity| entity.id()));
+        builder.extract_resources(world);
 
         builder.build()
     }
 
-    /// Write the dynamic entities and their corresponding components to the given world.
+    /// Write the resources, the dynamic entities and their corresponding components to the given world.
```

Comment:
Nit: I think we generally use the Oxford comma in the codebase (could be wrong)
```suggestion
    /// Write the resources, the dynamic entities, and their corresponding components to the given world.
```